src/scores/processing.py

```diff
@@ -288,15 +289,15 @@ def proportion_exceeding(
         that are greater than or equal to the corresponding threshold.
 
     """
-    return _binary_discretise_proportion(data, thresholds, ">=", preserve_dims, reduce_dims)
+    return _binary_discretise_proportion(data, thresholds, ">=", reduce_dims=reduce_dims, preserve_dims=preserve_dims)
 
 
 def _binary_discretise_proportion(
     data: XarrayLike,
     thresholds: Iterable,
     mode: str,
-    preserve_dims: FlexibleDimensionTypes = None,
     reduce_dims: FlexibleDimensionTypes = None,
+    preserve_dims: FlexibleDimensionTypes = None,
```

Comment:
Thanks for this - placing reduce first is consistent with other methods, I wish I had picked up on the discrepancy in the first place.